tap_process_street/schemas/approvals.json

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "createdDate": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "createdBy": {
+      "type": "string"
+    },
+    "updatedDate": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "updatedBy": {
+      "type": "string"
+    },
+    "approvalTask": {
+      "type": "string"
+    },
+    "subjectTask": {
+      "type": "string"
+    },
+    "status": {
+      "type": "string",
+      "enum": [
+        "Approved",
+        "Rejected"
+      ]
+    },
+    "reviewedBy": {
+      "type": "string"
+    },
+    "comment": {
+      "type": "string",
+      "nullable": true
```

Comment:
$please

Looks like we use `[null, string]` in some places and nullable here.  I don't feel strong about which we use but it would be good if we were consistent.

Reply:
right, I missed it